resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -51,32 +52,27 @@ import org.apache.spark.util._
 /**
  * Common application master functionality for Spark on Yarn.
  */
-private[spark] class ApplicationMaster(args: ApplicationMasterArguments) extends Logging {
+private[spark] class ApplicationMaster(
+    val args: ApplicationMasterArguments,
+    val sparkConf: SparkConf,
+    val yarnConf: YarnConfiguration)
+  extends Logging {
 
+  def this(sparkConf: SparkConf,
```

Comment:
See above constructor for multi-line args style.

Reply:
removed this constructor as part of below comment refactor